.github/workflows/unit-tests.yml

```diff
@@ -32,6 +32,12 @@ concurrency:
 jobs:
   cpu-tests:
     runs-on: ${{ matrix.os }}
+    env:
+      UV_SYSTEM_PIP: 1
+      CLEARML_NO_DEFAULT_SERVER: 1
+      CLEARML_API_HOST: "http://localhost"
+      CLEARML_WEB_HOST: "http://localhost"
+      CLEARML_FILES_HOST: "http://localhost"
```

Comment:
Why this vs `CLEARML_OFFLINE_MODE=1`, https://clear.ml/docs/latest/docs/guides/set_offline/#setting-task-to-offline-mode

https://github.com/pytorch/ignite/blob/5caecf25b467e4e2c246818ced24829aaca7b7d7/ignite/handlers/clearml_logger.py#L152-L167

Reply:
I don't quite understand that, but without 
      CLEARML_API_HOST: "http://localhost"
      CLEARML_WEB_HOST: "http://localhost"
      CLEARML_FILES_HOST: "http://localhost"
the following error is produced:
```  FAILED tests/ignite/contrib/engines/test_common.py::test_setup_clearml_logging - ValueError: ClearML configuration could not be found (missing `~/clearml.conf` or Environment CLEARML_API_HOST)```